include/fs/fs.h

```diff
@@ -67,6 +67,9 @@ enum {
  */
 #define FS_MOUNT_FLAG_AUTOMOUNT BIT(2)
 
+/** Force a format before attempting mount */
+#define FS_MOUNT_FLAG_FORCE_FORMAT BIT(3)
```

Comment:
Maybe take bit `7` for this (the underlying `flags` element is `uint8_t`) to reduce the chances of a future merge/rebase conflict if the Zephyr folks define new bits after bit `2` on their side?